presto-docs/src/main/sphinx/release/release-0.152.rst

```diff
@@ -6,11 +6,28 @@ General Changes
 ---------------
 
 * Add :func:`array_union` function.
+* Add :func:`reverse` function for arrays.
 * Fix issue that could cause queries with ``varchar`` literals to fail.
+* Fix query failure when referencing a field of a ``NULL`` row.
+* Improve query performance for multiple consecutive window functions.
+* Prevent web UI from breaking when query fails without an error code.
+* Display port on the task list in the web UI when multiple workers share the same host.
+* Add support for ``EXCEPT``.
+* Rename ``FLOAT`` type to ``REAL`` for better compatibility with the SQL standard.
 
 Hive Connector Changes
 ----------------------
 
 * Add file based security, which can be configured with the ``hive.security``
   and ``security.config-file`` config flags. See :doc:`/connector/hive-security`
   for more details.
+
+Verifier Changes
+----------------
+
+* Fix handling of shadow write queries with a ``LIMIT``.
+
+Local File Connector Changes
+----------------------------
+
+* Fix resource leak in local file connector.
```

Comment:
The "in local file connector" part is redundant, so this can be "Fix resource leak". However, it could be helpful to include more detail, such as what resource is leaked and under what conditions.

Maybe something like "Fix file descriptor leak when XXX fails."